config.json

```diff
@@ -244,7 +244,10 @@
     "restrict_to_login": false,
     "swatch_option_size": "22x22",
     "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none"
+    "social_icon_placement_bottom": "bottom_none",
+    "show_geotrust_ssl_seal": false,
+    "geotrust_ssl_seal_hostname": "",
```

Comment:
Since the trust ssl seal is for the store domain name. I don't see a reason why we should expose a field for the user to enter it unless there is some historic reason which I am not aware of.